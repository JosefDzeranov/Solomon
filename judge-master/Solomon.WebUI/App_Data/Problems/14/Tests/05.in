10
aaaaaaa
bbbbbbb
ccccccc
ddddddd
eeeeeee
aaaaaaa
bbbbbbb
ccccccc
ddddddd
eeeeeee