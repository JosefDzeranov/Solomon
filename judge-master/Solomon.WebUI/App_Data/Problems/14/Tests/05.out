35
aaaaaaa
aaaaaaa
bbbbbbb
bbbbbbb
ccccccc
ccccccc
ddddddd
ddddddd
eeeeeee
eeeeeee
